github-event-service/pull-request.ts

```diff
@@ -57,3 +51,26 @@ export async function handlePullRequestEvent(event: GithubPullRequestEvent) {
   }
 }
 
+/** 
+ * Creates a presubmit task for the pull request, if the application is enabled for
+ * the repo and the pull request has one of the target labels.
+ */
+async function createPresubmitTask(
+  repoId: string, pullRequestRef: firebase.firestore.DocumentReference) {
+    const repoDoc = await getConfig(repoId);
+    if (!repoDoc.exists) {
+      return;
+    }
+    const repoConfig = (await getConfig(repoId)).data() as BranchManagerRepoConfig;
```

Comment:
We aren't able to type `data()` it returns back a `DocumentData` object.  It would be nice if it was a generic, but I think that firebase doesn't do that because technically you can have different structures in documents in the same collection.

Reply:
I see, that makes sense but is also somehow unfortunate.